github-pages.gemspec

```diff
@@ -26,7 +26,8 @@ Gem::Specification.new do |s|
 
   s.add_dependency("mercenary", "~> 0.3")
   s.add_dependency("terminal-table", "~> 1.4")
-  s.add_dependency("nokogiri", ">= 1.8.2", "< 2.0") # CVE-2017-9050, CVE-2017-18258
+  # CVE-2017-9050, CVE-2017-18258, CVE-2018-14404
```

Comment:
Let’s actually just remove the comment, if you don’t mind. The Git history for this line will have the context :)

Reply:
lol, ok then :smile_cat: